.github/workflows/nightly-pipeline.yaml

```diff
@@ -46,11 +46,12 @@ jobs:
           ref: ${{ fromJSON(needs.get-run-info.outputs.obj).branch }}
           wait_interval: 120
           client_payload: ${{ toJSON(fromJSON(needs.get-run-info.outputs.obj).payloads.rmm) }}
-          propagate_failure: false
+          propagate_failure: true
           trigger_workflow: true
           wait_workflow: true
   rmm-tests:
     needs: [get-run-info, rmm-build]
+    if: ${{ needs.rmm-build.result == 'success' }}
```

Comment:
I think this is redundant. According to the docs below, the default conditional statement for jobs is `if: ${{ success() }}`.

- https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions

Reply:
👍🏿

Reply:
@AyodeAwe, please re-read that page. It says:

> A default status check of success() is applied unless you include one of these functions.

Therefore there's no reason to explicitly add `if: ${{ success() }}` since that's the default behavior.

Those lines should be removed.

Reply:
Sorry, yeah that's implicit. Will remove